### DISCUSSION TITLE
Login error after installation on a synology

### DISCUSSION THREAD

**Discussion by fabianbarben** [kimai/kimai#2910] [Questions & Answers] [answered]
Hy together

sorry for my English first ;-).

I tried to install Kimai2 on a synology using the ''official'' guide (thanks for that). I had some permission and a lot of ''not knowing enough'' issues. But now, it seems that I am close to a running system ;-).

But when I try to login, I get a message, that the user or the password is wrong. I createt many users on differet ways (console, directly in db, over self-registration), but every time the same result.

Any help would be very nice :-). Thanks in advance!

**Comment by kevinpapst** [comment, marked as answer]
Hey! Could be an encryption problem.
First: flush all cookies to get rid of any left-over. Then empty the session database with `TRUNCATE kimai2_sessions;` (just to make sure you really start from scratch.
Now: if you create the user as described in the guide from the command line with `bin/console ...` and you cannot login, but the user is in the database, it is likely that your bash PHP has different modules loaded than the web PHP.  

Kimai uses the best possible encryption method to store the passwords. Usually bcrypt or argon. I have seen some systems that had different PHP versions running - so make sure that PHP on the command line is the same as in web (both should be PHP 7.3, 7.4 or 8.0, but not different versions, that is extremely important!)

**Comment by fabianbarben** [comment]
Hello Kevin
Thank you for the fast answer and the right solution! I had to activate all the modules on the synology (Web Station) and it works ;-). Thanks a lot, you are great! My PHP-/Web-knowings are poor (as my English ;-)), but if I can help you otherwise (as ex. with a step-by-step tutorial how to install on a synology, let me know.
Have a good evening, Fabian

**Reply by kevinpapst** [reply]
Well, writing these docs is rather hard for me, because I know my way around these systems and especially Kimai pretty well.
So if you were missing something in the installation docs, it'll be great if you could share that and I will add it to the docs for the next person.
